commons/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="fr.free.nrw.commons"
+    package="free.nrw.commons"
```

Comment:
R is a generated class. Delete it and regenerate it.
A way to do this is to run mvn clean and mvn install.
mvn is described in the README.

I am at the embassy to meet the French Prime Minister :-)

Nicolas
On Oct 4, 2015 2:14 PM, "Josephine Lim" <notifications@github.com> wrote:

> In commons/AndroidManifest.xml
> <https://github.com/nicolas-raoul/apps-android-commons/pull/16#discussion_r41094512>
> :
>
> > @@ -1,5 +1,5 @@
> >  <manifest xmlns:android="http://schemas.android.com/apk/res/android"
> > -    package="fr.free.nrw.commons"
> > +    package="free.nrw.commons"
>
> I tried that initially but had a problem with it as it requires the
> addition of a 'fr' package in the middle of the package tiers and moving
> the 'free' package into it via refactor. The refactor seemed to break the
> links to the layout files, for example resDialogText =
> R.string.crash_dialog_text produces the error 'cannot resolve symbol R'.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nicolas-raoul/apps-android-commons/pull/16/files#r41094512>
> .
>